.env.example

```diff
@@ -0,0 +1,6 @@
+ENCRYPTION_KEY=
+DB_HOSTNAME=
+DB_USERNAME=
+DB_PASSWORD=
+DB_NAME=
+CPP_EXECUTABLE_PATH=/usr/bin/g++
```

Comment:
Please fix your editor's setting to automatically add newline at the end of files